CHANGELOG.mdx

```diff
@@ -16,6 +16,24 @@ https://github.com/RasaHQ/rasa/tree/main/changelog/ . -->
 
 <!-- TOWNCRIER -->
 
+## [3.3.0] - 2022-10-24                         
+### Improvements
+- [#11561](https://github.com/rasahq/rasa/issues/11561): Added option `--offset-timestamps-by-seconds` to offset the timestamp of events when using `rasa export`
+- [#11578](https://github.com/rasahq/rasa/issues/11578): Rasa supports native installations on Apple Silicon (M1 / M2). Please
+  follow the installation instructions and take a look at the limitations.
+- [#11596](https://github.com/rasahq/rasa/issues/11596): Caching `Message` and `Features` fingerprints unless they are altered, saving up to 2/3 of fingerprinting time in our tests.
+- [#11598](https://github.com/rasahq/rasa/issues/11598): Added package versions of component  dependencies as an additional part of fingerprinting calculation. Upgrading an dependency will thus lead to a retraining of the component in the future. Also, by changing fingerprint calculation, the next training after this change will be a complete retraining.
+- [#11623](https://github.com/rasahq/rasa/issues/11623): Export events continuously rather than loading all events in memory first when
+  using `rasa export`. Events will be streamed right from the start rather
+  than loading all events first and pushing them to the broker afterwards.
+
+### Bugfixes
+- [#11575](https://github.com/rasahq/rasa/issues/11575): Fix `BlockingIOError` when running `rasa interactive`, after the upgrade of `prompt-toolkit` dependency.
+
+### Improved Documentation
+- [#11571](https://github.com/rasahq/rasa/issues/11571): Rasa SDK documentation lives now in Rasa Open Source documentation under the _Rasa SDK_ category.
+
+
 ## [3.3.0a1] - 2022-10-10                           
```

Comment:
@m-vdb @ancalita should the alpha release details be present in the changelog or does this need to be removed?

Reply:
yeah feel free to remove

Reply:
To double-check, shouldn't the changelog entries that were against the alpha release now appear against the new minor?

Reply:
@ancalita you're right there were some duplications, the 3.3.0a1 entries were again duplicated in 3.3.0. I've done the cleaning needed, can you please also review 🙏